lib/timers.js

```diff
@@ -347,6 +353,9 @@ function tryOnTimeout(timer, start) {
         refCount--;
       timer[kRefed] = null;
 
+      if (timer[kHasPrimitive])
```

Comment:
I think @bnoordhuis mentioned at some point that it was a bit faster.

Reply:
I wonder what output turbolizer would give for this, this still doesn't seem right so I can only imagine it would be a polymorphism problem

Reply:
I don't think it's polymorphism. In the benchmark we don't call this with any values but timer objects. I'm reasonably certain it's the symbol because I've run into this a bunch while working on timers & http2.

Reply:
Oh whoops, I was thinking of the wrong function